test/functional/cypress/integration/google/accelerator_zones.spec.js

```diff
@@ -32,30 +37,25 @@ describe('google: Create Server Group Modal GPU Accelerators', () => {
   it('should show NVIDIA Tesla K80 after selecting us-east1-c zone', () => {
     cy.visit('#/applications/compute/clusters');
     cy.window().its('angular').should('exist');
-    cy.get('button:contains("Create Server Group")', { timeout: 10000 })
-      .should('be.visible')
-      .click();
+    cy.get('button:contains("Create Server Group")', { timeout: 10000 }).should('be.visible').click();
 
     // Select region and zone
     cy.get('v2-wizard-page[key=location]').within(() => {
       cy.get('div.form-group:contains("Region") select').select('us-east1');
     });
 
     cy.get('v2-wizard-page[key=zones]').within(() => {
-      cy.get('div.form-group:contains("Zone") select').select('us-east1-c');
+      cy.get('div.form-group:contains("Zone") select').select('us-east1-c', { force: true });
     });
 
-    cy.wait(1000);
-    cy.get('.btn-primary').first()
-      .should('be.visible')
-      .click({ force: true });  
+    cy.contains('ul.steps-indicator li a', 'Advanced Settings').click();
```

Comment:
Intentionally losing the primary button visible check?  And wait?

Reply:
Same case as noted here: https://github.com/spinnaker/deck/pull/10177#discussion_r2067836543